ui/views.py

```diff
@@ -21,6 +21,13 @@ def homepage(request):
             start_time__range=get_current_week()
         ).values_list("assigned_movie", flat=True)
     )
+
+    playing_movies_next_week = Movie.objects.filter(
+        pk__in=PlayingTime.objects.filter(
+            start_time__range=get_next_week()
+        ).values_list("assigned_movie", flat=True)
+    )
```

Comment:
Both of these above queries can be written as this as well:
`Movie.objects.filter(playing_time__start_time__range=get_X_week()).values_list("assigned_movie", flat=True)`
This way we should only have two queries, instead of four (not sure if this has been optimized in Django 4, though).